source/process/working-at-mattermost.rst

```diff
@@ -146,10 +150,11 @@ General Benefits
 Results matter more far more than hours in an office. Your time at Mattermost is flexible. Go see the dentist, visit a friend, walk your dog--no need to ask permission.
 
 If there's something urgent going on at work, just let teammates know your status.
+For most roles, we require the ability to join meetings between 8am and 1pm California time. The exception is people who are dedicated to support in other timezones, or team members that don't need to be in real-time discussions. 
 
-**2. Work from anywhere with a strong internet connection**
+**2. Work from anywhere (with good internet)** 
 
-The comfort of home, freedom to travel, no commute--learn more about our `remote work policies <https://docs.mattermost.com/process/working-at-mattermost.html#working-remotely>`_.
+The comfort of home, freedom to travel, no commute--learn more about our `work anywhere policies <https://docs.mattermost.com/process/working-at-mattermost.html#working-remotely>`_.
```

Comment:
I this this link needs to be updated since the title of the section earlier in the doc is updated. Would also be good to switch it to a relative link.

Reply:
@amyblais would you be able to update the link?

Reply:
@lfbrock I updated the link.